Community-Supported/publish-hyper/README.md

```diff
@@ -0,0 +1,26 @@
+# publish-hyper
```

Comment:
Don't you also need to add a link + short description to this example to the high-level README?